libs/dao/src/entities/delegation.entity.ts

```diff
@@ -0,0 +1,21 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryColumn } from 'typeorm';
+import { Dao } from './dao.entity';
+
+@Entity()
+export class Delegation {
+  @PrimaryColumn({ type: 'text', unique: true })
+  id: string;
+
+  @Column()
+  daoId: string;
+
+  @ManyToOne(() => Dao, (dao) => dao.delegations)
+  @JoinColumn({ name: 'dao_id' })
+  dao: Dao;
+
+  @Column()
+  accountId: string;
+
+  @Column()
+  balance: string;
```

Comment:
why not use bigint or numeric?

Reply:
That's not a big deal I guess since we're not calculating anything based on DB values - as an API response - it will be a string anyways. I'll change that.

Reply:
done.